### PR TITLE
[IMP] charts: set fixed font sizes in scorecard chart

### PR DESCRIPTION
**Pull request by anhe-odoo**
## Task Description

As a revamp of the scorecard design, we decided now to set the different font sizes to fixed value no matter how big the chart is, and to use adaptative font size when the scorecard is really too small to handle the fixed value.
This is a first step to a fully customizable scorecard where we will allow the user to choose the font sizes in the design pannel

## Related Task

- Task: 4176111

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_t("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: src/components/side_panel/chart/scorecard_chart_panel/scorecard_chart_design_panel.xml [hunk 1]
```diff
@@ -16,6 +16,16 @@
             onChange.bind="updateHumanizeNumbers"
           />
         </Section>
+        <Section class="'pt-0'">
```
Comment (hokolomopo) [inline src/components/side_panel/chart/scorecard_chart_panel/scorecard_chart_design_panel.xml:19]: Woudln't it be better to have a font size selector (like the one in the topbar) next to the other font options of the title (bold, italic, align ....) ?

File: src/components/figures/chart/scorecard/chart_scorecard.xml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <templates>
   <t t-name="o-spreadsheet-ScorecardChart">
-    <canvas class="o-figure-canvas w-100 h-100" t-ref="chartContainer" t-att-title="title"/>
+    <canvas
+      class="o-figure-canvas o-scorecard w-100 h-100"
```
Comment (hokolomopo) [inline src/components/figures/chart/scorecard/chart_scorecard.xml:4]: Unused class ATM.

File: tests/figures/chart/scorecard_chart_component.test.ts [hunk 3]
```diff
@@ -629,15 +614,15 @@ describe("Scorecard charts rendering", () => {
...
+    updateScorecardChartSize(50, 50);
     renderScorecardChart(model, chartId, sheetId, canvas);
 
     expect(scorecardChartStyle.baseline.fontSize).toBeLessThan(baselineFontSize);
     expect(scorecardChartStyle.title.fontSize).toEqual(titleFontSize);
     expect(scorecardChartStyle.key.fontSize).toBeLessThan(keyFontSize);
   });
 
-  test("Font size scale down if we put a long key value", () => {
+  test("Font sizes stay the same if we put a long key value", () => {
```
Comment (hokolomopo) [inline tests/figures/chart/scorecard_chart_component.test.ts:625]: This isn't true, you scale down the font size if the key value is very long

**Comment by pro-odoo** [comment]
robodoo r+

**Pull request closed without merging**